content/urbit-os/base/hood.md

```diff
@@ -20,37 +20,122 @@ Drum's additional API is as follows.
 
 A scry with a path of `/x/kiln/our` will return our ship as an `@p` with a `%noun` mark.
 
+Example:
+
+```
+> .^(@p %gx /=hood=/kiln/our/noun)
+~zod
+```
+
 #### Kernel update blocked?
 
 A scry with a path of `/x/kiln/lag` will return a `?` with a `%loob` mark specifying whether a kernel upgrade is blocked.
 
+Example:
+
+```
+> .^(? %gx /=hood=/kiln/lag/loob)
+%.n
+```
+
 #### Base hash
 
 A scry with a path of `/x/kiln/base-hash` will return the `%base` base hash as a `@uv` with a `%noun` mark.
 
+Example:
+
+```
+> .^(@uv %gx /=hood=/kiln/base-hash/noun)
+0v1l.6h9mk.s675m.jmi9q.g5isc.20q0l.g6r86.36v92.1l6db.li3hh.6r68a
```

Comment:
What did you run this scry on? I just booted a fakeship on 410k with Vere 3.4 and this doesn't work on my machine.

Reply:
I did it on my live ship. It seems if there's no mergebase it crashes. Theoretically it should just return 0v0 so it's a bug I guess. Should document as-is and fix in urbit/urbit